src/retrieve-validation/browser.ts

```diff
@@ -1,9 +1,29 @@
 // Imports
 import { W3CCSSValidatorResponse } from '.';
 import BadStatusError from './bad-status-error';
+import retrieveValidation from '.';
+import { W3CValidatorParameters } from '../types/parameters';
 
 // Utility function for retrieving response from W3C CSS Validator in a browser environment
-const retrieveInBrowser = async (url: string, timeout: number): Promise<W3CCSSValidatorResponse['cssvalidation']> => {
+const retrieveInBrowser: typeof retrieveValidation = async (
+	method: string,
```

Comment:
This should be more narrow:

```suggestion
	method: 'GET' | 'POST',
```